### DISCUSSION TITLE
［灌水］柚子社拿到了b站十万粉牌

### DISCUSSION THREAD

**Discussion by Signedinn** [share121/inter-knot#4801] [灌水]
就在今天柚子社发帖晒b站十万粉纪念牌
翻译：
“我们收到了来自bilibii的10 万粉丝突破
纪念牌。非常感谢！！”
![IMG_4549](https://github.com/user-attachments/assets/647f487b-7d5d-4009-9d65-9407dcdef749)

**Comment by March-777th** [comment]
柚子厨真是的<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Comment by ysdtal** [comment]
已阅，对着绫地宁宁打一角以示祝贺Ciallo～(∠・ω< )⌒☆​
